src/component/veilederverktoy/opprett-oppgave/components/oppgave-inner-form.tsx

```diff
@@ -47,7 +47,7 @@ function OppgaveInnerForm({
                     tema={tema}
                     veilederId={veilederId}
                     formikProps={formikProps}
-                    enhetId={!!enhetId ? enhetId : avsenderenhetId}
+                    enhetId={enhetId ? enhetId : avsenderenhetId}
```

Comment:
Denne er vel grei å beholde som den var? SonarCloud liker ikke endringen, så hvis vi ikke vil bruke `!!` så burde vi endre uttrykket

Reply:
Jeg fjernet !! fordi esLint liker ikke det. Jeg tror vi trenger ikke å endre uttrykke fordi javascript/typescript skal evaluere til true om variable er ikke (undefined, null, '', false, NaN)

Reply:
Okay, skjønner! Men for å blidgjøre SonarCloud også, så må vi gjøre om til noe sånt:
```suggestion
                    enhetId={enhetId ||  avsenderenhetId}
```